psychopy/experiment/components/_base.py

```diff
@@ -839,31 +839,6 @@ def writeFrameCodeJS(self, buff):
             buff.setIndentLevel(-1, relative=True)  # to exit the if block
             buff.writeIndented("}\n")
 
-    def writeRoutineEndCode(self, buff):
```

Comment:
I'm guessing you're removing this as sending events to ioHub is now done in the function which saves to PsychoPy?